src/setting.rs

```diff
@@ -133,6 +133,42 @@ impl SettingData {
     }
 }
 
+/// Payload of a `DEVSETTING.DAT` file (32 bytes).
+#[binrw]
+#[derive(Debug, PartialEq)]
+#[brw(little)]
+pub struct DevSetting {
```

Comment:
`DeviceSetting` is still pretty ambiguous though, since all `*SETTING.DAT` files are for devices. :-)

I think `MYSETTING.DAT`, `MYSETTING2.DAT` and `DJMMYSETTING.DAT` contain the options from the "My Settings" preferences page, and `DEVSETTING.DAT` contains the options from the "General" preferences page (and possibly other stuff).

Reply:
I think we can just keep the original name for now, then it's immediately apparent that it maps to the file with the same name.

We can later consider writing abstractions that read/write the files automatically and only expose a single settings object or something like that.